code/go/0chain.net/blobbercore/handler/handler_common.go

```diff
@@ -94,6 +94,27 @@ func HomepageHandler(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintf(w, "</br>")
 }
 
+type BlobberInfo struct {
+	ChainId string `json:"chainId"`
+	BlobberId string `json:"blobberId"`
+	BlobberPublicKey string `json:"blobberPubKey"`
+	BuildTag string `json:"buildTag"`
```

Comment:
Use underscore for json tags so that all APIs have the consistent response format.